web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -141,6 +127,55 @@ export const SidebarCardsList = (props: SidebarCardsListProps): ReactElement =>
     );
   };
 
+  const emptyForYouMessage = (): ReactNode => {
+    const showEmptyOpportunitiesMsg =
+      props.displayCertifications &&
+      props.displayFundings &&
+      props.certifications.length + props.fundings.length === 0;
+
+    const showCompleteRequiredTasksMsg =
+      !props.displayCertifications &&
+      !props.displayFundings &&
+      props.topCards.length + props.bottomCards.length === 0;
+
+    const isRemoteSellerWorker = (): boolean => {
+      if (business) {
+        const foreignBusinessType = determineForeignBusinessType(business.profileData.foreignBusinessTypeIds);
+        return foreignBusinessType === "REMOTE_SELLER" || foreignBusinessType === "REMOTE_WORKER";
+      }
+      return false;
+    };
+
+    const showEmptyForYouMessage =
+      showEmptyOpportunitiesMsg || showCompleteRequiredTasksMsg || isRemoteSellerWorker();
+
+    const getTopText = (): string => {
+      if (isRemoteSellerWorker()) return Config.dashboardDefaults.emptyOpportunitiesRemoteSellerWorkerText;
+      if (showCompleteRequiredTasksMsg) return Config.dashboardDefaults.completeRequiredTasksText;
+      if (showEmptyOpportunitiesMsg) return Config.dashboardDefaults.emptyOpportunitiesHeader;
+      return "";
+    };
+
+    const getBottomText = (): string => {
+      if (showEmptyOpportunitiesMsg) return Config.dashboardDefaults.emptyOpportunitiesText;
+      return "";
+    };
+
+    return (
+      <>
+        {showEmptyForYouMessage && (
```

Comment:
dust - would it make sense to add a guard clause for `!showEmptyForYouMessage`

Reply:
I thought we landed on this `&&` pattern, didn't we? It's kindof behaving like a guard clause and the alternative would be 2 return statements, one with an empty fragment and the other being the actual content.

Reply:
Based on how this logic is written, if `showEmptyForYouMessage` is false a react fragment is returned, but if `showEmptyForYouMessage` is true, the same react fragment is returned with additional children elements.

The guard clause would separate the logic on line 165 from 166
```
if (!showEmptyForYouMessage) return null
return <div>...
```

Reply:
Right, I thought in this instance we opted for the && pattern for instances like this. I can go either way.

Reply:
ok, I'm good either way as well